lib/checks.js

```diff
@@ -2,12 +2,14 @@
  * Copyright (c) 2012 Mathieu Turcotte
  * Licensed under the MIT license.
  */
+'use strict';
 
 var util = require('util');
 
 var errors = module.exports = require('./errors');
 
 function failCheck(ExceptionConstructor, callee, messageFormat, formatArgs) {
+    /* jshint validthis: true */
```

Comment:
Could probably pass null to avoid the problem in the first place, would get the global this. It doesn't matter in that situation. I don't like to leave comments to please a tool that is not smart enough to understand what's going on.